src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -388,100 +402,251 @@ private SyntaxNode GetNodeContainingTargetNode(SyntaxNode matchingNode)
                     return nameMemberCref;
                 }
 
-                var newParameters = PermuteDeclaration(nameMemberCref.Parameters.Parameters, signaturePermutation);
+                var parameterList = nameMemberCref.Parameters;
+                var newParameters = PermuteDeclaration(parameterList.Parameters, parameterList.OpenParenToken, signaturePermutation);
 
-                var newCrefParameterList = nameMemberCref.Parameters.WithParameters(newParameters);
+                var newCrefParameterList = parameterList.WithParameters(newParameters).WithOpenParenToken(parameterList.OpenParenToken.WithTrailingTrivia());
                 return nameMemberCref.WithParameters(newCrefParameterList);
             }
 
             Debug.Assert(false, "Unknown reference location");
             return null;
         }
 
-        private SeparatedSyntaxList<T> PermuteDeclaration<T>(SeparatedSyntaxList<T> list, SignatureChange updatedSignature) where T : SyntaxNode
+        private SeparatedSyntaxList<T> PermuteDeclaration<T>(
+            SeparatedSyntaxList<T> list,
+            SyntaxToken openParamToken,
```

Comment:
Did you mean 'openParen' instead of 'openParam'?  not sure if this was intentional.